src/AppMenu.tsx

```diff
@@ -189,6 +193,33 @@ class AppMenu extends Component<Props, State> {
               </ListItemIcon>
               <ListItemText primary="Export Diagnostic Report"></ListItemText>
             </ListItemButton>
+            <Tooltip
+              disableInteractive
+              title="Choreo can now output java files containing variables and constants defined in the GUI."
```

Comment:
```suggestion
              title="Output Java files containing variables and constants defined in the GUI."
```
The `now` phrasing will age poorly, since the feature won't be new forever.